services.yaml

```diff
@@ -1437,6 +1437,71 @@ paths:
           $ref: "#/responses/UnauthorizedError"
         404:
           $ref: "#/responses/NotFoundError"
+  /services/{env_id}/settings/publish-metrics:
+    get:
+      tags:
+        - Publish Metrics
+      summary: "Fetch Publish Metrics settings"
+      description: "Use this call to fetch data about publish metrics settings, output format and the actual metrics which are currently beeing published"
+      parameters:
+        - in: path
+          name: env_id
+          description: Environment to use
+          type: string
+          required: true
+      responses:
+        200:
+          description: Current settings for publish metrics
+          schema:
+            $ref: "#/definitions/PublishMetricsSettings"
+        401:
+          $ref: "#/responses/UnauthorizedError"
+        404:
+          $ref: "#/responses/NotFoundError"
+    post:
+      tags:
+        - Publish Metrics
+      summary: "Set Publish Metrics"
+      description: "Use this call to set and customize publish metrics. Use one of the presets, or customize you tokens by setting output_format to 'custom'."
+      parameters:
+        - in: path
+          name: env_id
+          description: Environment to use
+          type: string
+          required: true
+        - in: body
+          name: settings
+          desciption: Publish metrics settings
+          schema:
+            $ref: "#/definitions/PublishMetricsSettings"
+      responses:
+        200:
+          $ref: "#/responses/OK"
+        401:
+          $ref: "#/responses/UnauthorizedError"
+        404:
+          $ref: "#/responses/NotFoundError"
+  /services/{env_id}/settings/publish-metrics/available_formats:
+    get:
+      tags:
+        - Publish Metrics
+      summary: "Fetch Available formats"
+      description: "Use this call to fetch available formats / presets, their metrics and their tokens"
+      parameters:
+        - in: path
+          name: env_id
+          description: Environment to use
+          type: string
+          required: true
+      responses:
+        200:
+          description: List of Available formats and metrics
+          schema:
+            $ref: "#/definitions/MetricFormats"
```

Comment:
i would rename this as PublishMetricFormats

Reply:
done